### DISCUSSION TITLE
在自駕車上裝上一對圓滾滾的大眼睛

### DISCUSSION THREAD

**Discussion by Tuoresa** [1101-GS4719-C/Python-Community-GS4719-C#1047] [General]
「根據研究人員所述，一對在無人車前旋轉的眼睛，
可以讓站在路邊的人更清楚他們是否有被看見。當人們面對迎面而來的交通，決定過馬路的完美時刻時，這會是很有用的資訊。」

擁有大眼睛的無人車如果滿街跑感覺好像什麼動畫場景哈哈！
不過看到這篇文章的時候感覺這樣的提示方式蠻神奇的，
實驗結果中的男女差異也很有趣。

雖然現在試驗仍在虛擬實境階段，
仍希望自駕車的安全性能真正獲得提高。

不過 過馬路安全與否除了車輛因素之外
有時候感覺也跟行人自己的習慣及安全意識有關
(不管三七二十一任意穿越馬路之類......

https://tomorrowsci.com/technology/%e5%b7%a5%e7%a8%8b%e5%b8%ab%e5%80%91%e5%9c%a8%e8%87%aa%e9%a7%95%e8%bb%8a%e4%b8%8a%e8%a3%9d%e4%b8%8a%e4%b8%80%e5%b0%8d%e7%9c%bc%e7%9d%9b%ef%bc%8c%e7%9b%ae%e7%9a%84%e6%98%af%e8%ae%93%e6%9c%aa%e4%be%86/

**Comment by Twenty0522** [comment]
我覺得這可以讓老人更加安心的過馬路而且外型可愛，但可能會讓小孩被眼睛吸引而停在馬路中間，雖然設計很可愛，但還需要更加縝密的想法。

**Comment by harrylee1971** [comment]
平常開車路上有很多三寶，或許測試結果是有助於行人安全，但真的會撞到的三寶估計有這個也沒用處

**Comment by s96015welson** [comment]
我認為這項技術多少能提高安全性，而且外觀滿可愛的。不過再厲害的機器與科技仍然無法做到萬無一失，只要還有發生意外的可能，肇事責任還是無法劃分清楚，而這可能才是自駕車真正過不去的檻。

**Comment by HsuChen88** [comment]
這雙眼睛對其他路人一定有一寫幫助，不過我相信大部分人在夜深人靜的時候會被這對眼睛嚇死。

**Comment by 110302550** [comment]
自動駕駛我覺得真的會改善人類很大的生活，如果能讓自動駕駛更加完善的話，可能之後都不用司機了，還不會因為打瞌睡造成的車禍，能優化人類的生活品質甚至是安全問題

**Comment by ghost** [comment]
我認為這項技術很有趣，透過擬人的眼睛外觀，來增強安全性。不過我覺得自駕車的問題在於發生意外時，肇事責任無法劃分清楚，這仍是需要克服的一大難題

**Comment by wenchi9023** [comment]
人們正常都會在綠燈時才會過馬路，此時注視車的用處不大；僅有在黃燈或者無紅綠燈的狀態下，人們才有可能需要注意來車，這時注視車確實能夠幫助行人判斷是否可以跨越馬路。但仔細一想的話會發現，注視車的存在是否潛在地助長不看交通的行人增加呢? 這項設計新穎且且確實讓行人在過馬路時更加注意，但感覺還是需要更多的評估
